installinator/src/dispatch.rs

```diff
@@ -126,6 +126,11 @@ struct InstallOpts {
     #[command(flatten)]
     artifact_ids: ArtifactIdOpts,
 
+    /// If true, perform sled-agent-like bootstrapping operations on startup
+    /// (e.g., configure and enable maghemite).
+    #[clap(long)]
+    bootstrap_sled: bool,
```

Comment:
In a production environment, we expect this to basically always be "true", right?

Reply:
Yep: the command in `manifest.xml` unconditionally passes this flag. This is only here to allow installiantor to run by hand / in tests.